### PR TITLE
Fix gl2d cross / square marker symbol offset

### PR DESCRIPTION
**Pull request by etpinard**
Including:

- https://github.com/gl-vis/gl-scatter2d-sdf/compare/6014a95c9e1b6891034dd8cf17f9d946e8a1d5ba...master
- and https://github.com/gl-vis/gl-scatter2d-sdf/pull/3
- https://github.com/dfcreative/font-atlas-sdf/commit/f52d72270648905fea5de746117686c13589058c
- https://github.com/gl-vis/gl-scatter2d-sdf/commit/1cd0e334aef67923a6e2bec3fd0b34a23a6896b5

### CODE REVIEW

**Comment by etpinard** [comment]
@dfcreative something is still off:

peek 2017-04-27 15-33

**Comment by dfcreative** [comment]
I don't see that with gl-scatter2d-sdf@1.3.7, here you use gl-scatter2d-sdf@1.3.4

File: test/image/mocks/gl2d_marker_symbols.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      "y": [1, 1, 1, 1, 1, 1, 1],
+      "mode": "markers",
+      "text": [
+        "marker symbol: 'circle'",
+        "marker symbol: 'circle-open'",
+        "marker symbol: 'square'",
+        "marker symbol: 'square-open'",
+        "marker symbol: 'diamond'",
+        "marker symbol: 'diamond-open'",
+        "marker symbol: 'cross'"
```
Comment (etpinard) [inline test/image/mocks/gl2d_marker_symbols.json]: @dfcreative looking good :ok_hand:

Comment (etpinard) [inline test/image/mocks/gl2d_marker_symbols.json]: Would you mind publishing `gl-scatter2d-sdf@1.3.9` to npm?

Comment (dfcreative) [inline test/image/mocks/gl2d_marker_symbols.json]: Done!

**Pull request merged**